lib/fizzbuzz.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'pry'
+class FizzBuzz
+
+  def initialize(number)
+    @number =  number
+  end
+  def divisible_by_three
+    if @number % 3 == 0
+    "Fizz"
+    else
+      "This number is not divisible by three"
+    end
```

Comment:
You could change this to be something like:
```
return "Fizz" if @number % 3 == 0

"This number is not divisible by three"
```

Both are fine, i think it's more of a style preference

Reply:
That is also changed! :)